addons/website_form/static/src/snippets/s_website_form/000.js

```diff
@@ -21,11 +23,20 @@ odoo.define('website_form.s_website_form', function (require) {
          */
         init: function () {
             this._super(...arguments);
+            this._recaptcha = new ReCaptcha(this);
```

Comment:
You give `this` to the ReCaptcha constructor while it takes no argument (same everywhere)